src/codewhisperer/tracker/codewhispererTracker.ts

```diff
@@ -5,10 +5,10 @@
 
 import * as vscode from 'vscode'
 import globals from '../../shared/extensionGlobals'
-import * as telemetry from '../../shared/telemetry/telemetry'
 import { distance } from 'fastest-levenshtein'
 import { AcceptedSuggestionEntry } from '../models/model'
 import { getLogger } from '../../shared/logger/logger'
+import { telemetry } from '../../shared/telemetry/spans'
```

Comment:
Might be able to merge the telemetry object with the types to keep the `import * as telemetry` pattern. Potentially use a namespace?

Reply:
would be nice ergonomics, but perhaps not worth it if it requires a namespace